tests/cypress/integration/actions_users/registration_involved/case_4_assign_taks_job_users.js

```diff
@@ -70,39 +86,93 @@ context('Multiple users. Assign task, job.', () => {
             );
             cy.logout(thirdUserName);
         });
-        it('First user login and create a task', () => {
+        it('First user login, create a task and logout', () => {
+            if (Cypress.browser.family !== 'chromium') {
+                cy.get('.cvat-modal-unsupported-platform-warning').within(() => {
+                    cy.contains('button', 'OK').click();
+                });
+            }
             cy.login();
             cy.imageGenerator(imagesFolder, imageFileName, width, height, color, posX, posY, labelName, imagesCount);
             cy.createZipArchive(directoryToArchive, archivePath);
             cy.createAnnotationTask(taskName, labelName, attrName, textDefaultValue, archiveName);
+            cy.logout();
+        });
+        it('Second user login, tries to add label and logout', () => {
+            cy.login(secondUserName, secondUser.password);
+            cy.openTask(taskName);
+            cy.addNewLabel('failAddLabel');
+            cy.get('.cvat-notification-notice-update-task-failed').should('exist');
+            cy.closeNotification('.cvat-notification-notice-update-task-failed');
```

Comment:
The same comment about closeNotification like in PR with projects.

Reply:
Unnecessary actions removed.